### PR TITLE
Add tilt servo setup page; add note re: binary servo rate

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

**Comment by Hwurzburg** [comment]
oops a file got in that shouldnt have...will fix

File: plane/source/docs/tilt-rotor-tips.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- Now we will adjust travel. Its an iterative process, changing the value in MANUAL mode for one endpoint (min or max, depending on servo direction), then changing to QSTABLIZE and adjusting the other endpoint . The Servo Output screen is a "live screen".When you change it, it's value is instantly reflected in servo output. You will want to be sure that the MANUAL mode position places both motors against the stops (if used), in full forward flight position with just a little bit of torque...too much and the servo will overheat, too little and you get a loose position which can bounce a bit in the air. You want both motors EXACTLY aligned with props back on...a difference in fixed wing mode will result in roll force. Enough miss-alignment and you wont be able to stop the roll with full aileron!
```
Comment (tridge) [inline plane/source/docs/tilt-rotor-tips.rst:21]: stray space before . and ... between words

File: plane/source/docs/tilt-rotor-tips.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- In QSTABILIZE, both motors should be aligned. They can be vertical, but you can set them up such that when the AHRS horizon is level, they are. This results in them pointing a little forward when the airframe is flat (and the horizon will be 2-5 deg negative since you are not in flying attitude)....read https://ardupilot.org/plane/docs/quadplane-tips.html for more information
```
Comment (tridge) [inline plane/source/docs/tilt-rotor-tips.rst:22]: .... looks wrong

Comment (Hwurzburg) [reply plane/source/docs/tilt-rotor-tips.rst:22]: fixed

File: plane/source/docs/tilt-rotor-tips.rst (hunk 1 above)
Comment (Hwurzburg) [reply plane/source/docs/tilt-rotor-tips.rst]: fixed

File: plane/source/docs/tilt-rotor-tips.rst [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+=====================
+Tilt Rotor Setup Tips
+=====================
+
+Once you have the servo function assignment completed, and other parameters discussed in :ref:`guide-tilt-rotor` entered, you can begin to setup the tilt mechanism's travel and range.
+
+
+In Mission Planner's SETUP (or Initial Setup depending on release version) menu, under Mandatory Hardware/Servo Outputs you will see each outputs assignment, min/max/trim, and reverse. You will be switching back and forth between this tab and the DATA (previously Flight Data) tab several times.
+
+.. tip:: Miss-alignments as the motors transition to/from forward flight can have a big impact on smoothness or even airworthiness. It is worthwhile to the get them synchronized and aligned as precisely as possible.
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply plane/source/docs/tilt-rotor-tips.rst]: fixed

**Pull request merged**